features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/modules/statistics/usage.jag

```diff
@@ -273,6 +277,7 @@ function getSubscriberCountByAPIs(providerName) {
             };
 
         } else {
+            usage=JSON.parse(gson.toJson(usage));
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea